README.Rmd

```diff
@@ -33,6 +33,10 @@ and outcomes that these two countries have faced during the pandemic. However,
 further analysis is required to better understand the differences present in
 the Canada-US responses to COVID-19. 
 
+# Report
+
+The final report can be found [here](https://github.com/UBC-MDS/covid-19-cases-vs-tests-analysis/blob/main/doc/covid-response.pdf)
+
 # Usage
```

Comment:
Are we missing "os" and "calendar" packages that are needed in download.py and eda_codiv19.py?